stories/8-Toast.stories.tsx

```diff
@@ -0,0 +1,157 @@
+import "./utils/constants/toast/_toast.scss";
+
+import {storiesOf} from "@storybook/react";
+import React from "react";
+
+import Button from "../src/button/Button";
+import {useToaster} from "../src/toast/util/toastHooks";
+import StoryFragment from "./utils/StoryFragment";
+import Toast from "../src/toast/Toast";
+
+function ToastComponent() {
+  const {display, update, hideAll} = useToaster();
+
+  return (
+    <StoryFragment>
+      <div className={"toast-button-group"}>
+        <Button
+          type={"button"}
+          onClick={() =>
+            display({
+              render: () => {
```

Comment:
Same for others below too
```suggestion
              render() {
```